rulebook.tex

```diff
@@ -763,315 +977,132 @@ \subsubsection{Markers}
 one will be placed on the input, another one on the output side. All
 markers used are depicted in \reftab{tab:markers}. The marker will be
 horizontally centered below the conveyor belt. The vertical distance
-(between tag and conveyor belt) will be about
+between the tag's upper edge and the conveyor belt will be about
 \SI{35}{\centi\metre}. The markers will be mounted at the same
-position on all machines in a best effort fashion. But teams should
-expect to calibrate individually per marker. They are available for
-printing on the RoboCup Logistics website.
-
+position on all machines in a best effort fashion, but teams should expect
+slight deviations to their exact position.
+The markers are available for printing on the rulebook github\footnote{
+  \url{https://github.com/robocup-logistics/rcll-rulebook/tree/master/markers/numbered_markers.pdf} % chktex ignore-long-line
+}.
 
-\subsubsection{Machine Positioning}
-\label{sec:machine-swapping}
-The zones and rotations for the MPS will be randomly chosen by the RefBox.
-Each game will have a new randomly generated field layout, the referees will
-place the MPS in the middle of the zone with given rotation at there best effort
-but errors of the positioning should be expected.
+\subsection{Mockup Machines}
```

Comment:
Maybe we call the section before Physical Machines and Machines is the Chapter?

Reply:
That does not make sens to me as you need to know what the actual machines look like before you can talk about using mocked-up versions of them.